### PR TITLE
Improve documentation

### LINKED ISSUE #80: No documentation of - and + except in issue or git log
Please add a description about the - and + indicators in the "Available packages" overview to the man page.

### PR DESCRIPTION
**Pull request by alexanderkogan**
Fixes #80

### CODE REVIEW

**Comment by alexanderkogan** [comment]
Did you convert the md somehow to the man page or should I edit it manually?

**Comment by pbrisbin** [comment]
`make man` will update the groff content from the markdown. It requires a `pandoc` on `$PATH`. If you'd like I can run that part for you after merge / before release.

**Comment by alexanderkogan** [comment]
> If you'd like I can run that part for you after merge / before release.
That would be nice.

I had some merge problems, hence the merge commit, but I think it will be fine, if you just squash before merging the PR.

File: doc/downgrade.8.md [hunk 1]
```diff
@@ -10,6 +10,55 @@
...
+  \+ indicates the currently installed version.
+
+*enumeration*
+  An enumeration of the entries for selection.
+
+*package name*
+  The name of the package.
+
+*package version*
+  The version of the package as noted in the A.L.A.
```
Comment (pbrisbin) [inline doc/downgrade.8.md]: I would consider dropping (or rewording) "as noted in the A.L.A." since that doesn't apply to any rows that are `(local)`.

**Comment by pbrisbin** [comment]
Released in v6.2.2. Thanks!

**Pull request merged**